pkg/mover/chart.go

```diff
@@ -255,13 +256,55 @@ func (cm *ChartMover) loadChartFromIntermediateBundle(bundlePath string) error {
 
 // loadChartFromPath load the chart in memory from a given path
 func (cm *ChartMover) loadChartFromPath(path string) error {
-	var err error
-	if cm.chart, err = loader.Load(path); err != nil {
-		return &ChartLoadingError{Path: path, Inner: err}
+	// always unpack the chart tarball to a directory to avoid issues loading
+	// files if the tgz contains replicated entries
+	dir := path
+	info, err := os.Stat(path)
+	if err != nil {
+		return &ChartLoadingError{Path: dir, Inner: err}
+	}
+	if !info.IsDir() {
```

Comment:
This seems like a good candidate to its own function.

Reply:
yep, the only caveat is that defer to remove the tempdir... but I will find a way.

Reply:
think it is now much better